x/nft/go.mod

```diff
@@ -27,6 +27,7 @@ require (
 require (
 	cosmossdk.io/collections v0.4.0 // indirect
 	cosmossdk.io/x/auth v0.0.0-00010101000000-000000000000
+	cosmossdk.io/x/distribution v0.0.0-20230925135524-a1bc045b3190 // indirect
```

Comment:
Why do all the above modules now include a dependency on x/distribution?

Reply:
+1 we should remove this if possible

Reply:
removed